### ISSUE TITLE
nutups reports driver.parameter.pollfreq instead of AC frequency

### ISSUE THREAD

**Issue by fastcat** [munin-monitoring/munin#1164]
**Describe the bug**
The `nutups_` plugin has a too-broad regex match when processing the output of `upsc`, and as a result will report the value of `driver.parameter.pollfreq` as the "AC frequency", instead of `input.frequency` or `output.frequency`

**To Reproduce**
1. Enable `nutups_foo_freq`
2. Watch the reported values
3. Observe that it always reports a dead constant 30.0, not the actual input or output AC frequency from the UPS

**Expected behavior**

**Screenshots & Logs**
Example output of upsc on my system searching for the `/^[^:]*freq/` regex the plugin uses:
```
driver.parameter.pollfreq: 30
input.frequency: 59.9
output.frequency: 59.9
output.frequency.nominal: 60
```
The `sed` invocation the plugin uses (https://github.com/munin-monitoring/munin/blob/master/plugins/node.d/nutups_#L92) transforms this into:
```
acfreq.value 30
acfreq.value 59.9
acfreq.value 59.9
acfreq.value 60
```
And it takes the first value from that, which sadly has nothing to do with AC line frequencies.

**Desktop (please complete the following information):**
 - OS+Distribution Version: Debian 10 (Buster)
 - Munin Version 2.0.45-1

**Additional context**
Documentation from NUT on the standard UPS variables is here: https://networkupstools.org/docs/user-manual.chunked/apcs01.html

It might make sense to search specifically for the `input.frequency`, `output.frequency`, and possibly their related `.nominal`, `.low`, and `.high` properties to graph.

**Comment by sumpfralle** [comment]
Thank you for reporting this issue!

Could you please propose a changed regular expression that would fix the issue?
(I do not have a UPS device around)

**Comment by sumpfralle** [comment]
Fixed in #1166, thanks!
